azkaban-common/src/main/java/azkaban/executor/ExecutorLoader.java

```diff
@@ -238,14 +238,14 @@ List<Pair<ExecutionReference, ExecutableFlow>> fetchQueuedFlows()
       throws ExecutorManagerException;
 
   /**
-   * This method is used to get flows fetched in Queue. Flows can be in queue in ready, dispatching
+   * This method is used to get flow ids fetched in Queue. Flows can be in queue in ready, dispatching
    * or preparing state while in queue. That is why it is expecting status in parameter.
    *
    * @param status
    * @return
    * @throws ExecutorManagerException
    */
-  List<Pair<ExecutionReference, ExecutableFlow>> fetchQueuedFlows(Status status)
+  List<Integer> selectQueuedFlows(Status status)
```

Comment:
nit: maybe instead of `select...` we can name the new methods to `fetch...IDs`

Reply:
I personally think so....
But in *DAO.java, we use the term "select" to select only execId/id field and term "fetch" to fetch the whole row. Wanna keep that convention.